src/libraries/System.DirectoryServices.AccountManagement/tests/PrincipalTest.cs

```diff
@@ -15,8 +15,8 @@ public abstract class PrincipalTest : IDisposable
 
         private void RefreshContext()
         {
-            string username = "Administrator";
-            string password = "Adrumble@6";
+            // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Unit test dummy credentials.")]
+            string username = "Administrator", password = "Adrumble@6";
```

Comment:
Does changing Adrumble@6 to 'password' still allow the test to pass, and mean you can eliminate the suppression? If so I think it would be better to do that

Reply:
I don't feel confident about this change. I discusses similar change in System.Security.Cryptography with @bartonjs and he prefer not to refactor passwords. I did the refactoring only in System.Net.* area as I'm member of it.

Reply:
Ok